docs/pages/trustedclusters.mdx

```diff
@@ -191,21 +180,21 @@ Now, the administrator of "leaf" must create the following resource file:
 kind: trusted_cluster
 version: v2
 metadata:
-  # the trusted cluster name MUST match the 'cluster_name' setting of the
+  # The trusted cluster name MUST match the 'cluster_name' setting of the
   # root cluster
   name: root
 spec:
-  # this field allows to create tunnels that are disabled, but can be enabled later.
+  # This field allows to create tunnels that are disabled, but can be enabled later.
   enabled: true
-  # the token expected by the "root" cluster:
+  # The token expected by the "root" cluster:
   token: ba4825847f0378bcdfe18113c4998498
-  # the address in 'host:port' form of the reverse tunnel listening port on the
+  # The address in 'host:port' form of the reverse tunnel listening port on the
   # "root" proxy server:
   tunnel_addr: root.example.com:3024
-  # the address in 'host:port' form of the web listening port on the
+  # The address in 'host:port' form of the web listening port on the
   # "root" proxy server:
   web_proxy_addr: root.example.com:3080
```

Comment:
```suggestion
  web_proxy_addr: root.example.com:443
```

I'm going to make proxy addr, 443 since this is the default for cloud hosted Teleport

Reply:
Sure, 443 is likely the default port. Also, I just reviewed the outcome of this: https://github.com/gravitational/teleport/issues/4924 and don't believe any changes are absolutely necessary elsewhere since 6.2+ each likely port will be tried. 

Going forward, I will try to adjust the default port to 443 where appropriate.